tutorials/clinical/10_ieeg_localize.py

```diff
@@ -111,90 +113,38 @@ def plot_overlay(image, compare, title, thresh=None):
 T1 = nib.load(op.join(misc_path, 'seeg', 'sample_seeg_T1.mgz'))
 CT_orig = nib.load(op.join(misc_path, 'seeg', 'sample_seeg_CT.mgz'))
 
-# resample to T1 shape
+# resample to T1's definition of world coordinates
 CT_resampled = resample(moving=CT_orig.get_fdata(),
                         static=T1.get_fdata(),
                         moving_affine=CT_orig.affine,
                         static_affine=T1.affine,
                         between_affine=None)
 plot_overlay(T1, CT_resampled, 'Unaligned CT Overlaid on T1', thresh=0.95)
+del CT_resampled
 
 ###############################################################################
-# Now we need to align our CT image to the T1 image.
+# Now we need to align our CT image to the T1 image. Here we use (but do not
+# execute because it's slow) the call:
 #
 # .. code-block:: python
 #
-#     # normalize intensities
-#     mri_to = T1.get_fdata().copy()
-#     mri_to /= mri_to.max()
-#     ct_from = CT_orig.get_fdata().copy()
-#     ct_from /= ct_from.max()
+#    affine, _ = compute_volume_registration(CT_orig, T1, pipeline='rigid')
```

Comment:
This should be `rigids` and it doesn't give a good registration. It's not terribly off but it's not good enough. I'm not sure exactly why but I would guess from my testing that the zooms for the translation actually helped. Maybe if it's translated with a lot of optimization it can get stuck in a local minimum, not sure just thinking out loud.

Reply:
Also this took more than 10 minutes on my macbook, I think because of the translation and that it didn't work properly so then the rigid optimization took longer.

Reply:
> This should be rigids and it doesn't give a good registration. It's not terribly off but it's not good enough. I'm not sure exactly why but I would guess from my testing that the zooms for the translation actually helped. Maybe if it's translated with a lot of optimization it can get stuck in a local minimum, not sure just thinking out loud.

Can you push a line that does work after I make a commit where `zooms` can be specified at each level?